### PR TITLE
added flames game in python

### LINKED ISSUE #5: Flames Game
# Description

This is a fun game. Come up with a program that displays the relationship between two persons. Read more about it [here](https://www.wikihow.com/Play-%22Flame%22)

## Language

- [x] C++
- [x] Java: @Bhaswati-Saha
- [x] Python: @silentwraith6 

## Type of issue

- [x] Feature
- [ ] Bug

### PR DESCRIPTION
**Pull request by silentwraith6**
# Title/Heading

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #5 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generates no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by sakshamarora1** [comment]
Thanks for making a Pull Request @silentwraith6. Please go through the [Contributing Guidelines](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/CONTRIBUTING.md) (You should create a markdown file and add the snippet in the format specified in the guidelines) and make the changes accordingly! :D

Also don't forget to edit the PR description according to your changes 👍

**Comment by silentwraith6** [comment]
Changed it according to guidelines

File: snippets/python/flamesGame.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+firstName = [i for i in firstName] 
+secondName = [j for j in secondName] 
+
+flame(firstName, secondName) 
+```
+
+## Input
+Ajay
+Priya
+
```
Comment (sakshamarora1) [inline snippets/python/flamesGame.md]:
```
Ajay
Priya
```

File: snippets/python/flamesGame.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Output
+Ajay is FRIEND of Priya
```
Comment (sakshamarora1) [inline snippets/python/flamesGame.md]:
```
Ajay is FRIEND of Priya
```

File: snippets/python/flamesGame.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# This is a fun game. Come up with a program that displays the relationship between two persons.
```
Comment (kaustubhgupta) [inline snippets/python/flamesGame.md]: Edit this title, remove "come up". Also rewrite the heading to convey the same thing

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Great! It'd be great if you could add the input and output in ``` block. Then it will be ready for merging!

File: snippets/python/flamesGame.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+## Code
+
+```python
+def flame(firstName, secondName): 
+	l, sc = 1, 0
```
Comment (kaustubhgupta) [inline snippets/python/flamesGame.md]: Can you use better variable names here?

File: snippets/python/flamesGame.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+		print(q, "is going to MARRY", w) 
+	elif (flamesString[0] == 'l'): 
+		print(q, "is in LOVE with", w) 
+	elif (flamesString[0] == 'a'): 
+		print(q, "is more AFFECTIONATE towards", w) 
+	else: 
+		print(q, "and", w, "are SIBLINGS ") 
+
+# Driver code 
+firstName = input("Enter your name\n")
```
Comment (kaustubhgupta) [inline snippets/python/flamesGame.md]: Put this driver code in if name==

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by silentwraith6** [comment]
Now please check if its ok?

File: snippets/python/flamesGame.md (hunk 1 above)
Comment (sakshamarora1) [reply snippets/python/flamesGame.md]: I think I wasn't clear, I meant add the input and output in their separate blocks.[Example](https://github.com/Developer-Students-Club-MAIT/cool_code_snippets/blob/master/snippets/python/tablularformatting.md)

Comment (silentwraith6) [reply snippets/python/flamesGame.md]: Ok , understood now, will change that accordingly

Comment (silentwraith6) [reply snippets/python/flamesGame.md]: Done the required changes. Please check

File: snippets/python/flamesGame.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+## Input
+Ajay
+Priya
+```
+
+```
+## Output
+Relationship status :Friends
+```
+
```
Comment (sakshamarora1) [inline snippets/python/flamesGame.md]:
~~~suggestion
## Input
```
Ajay
Priya
```

## Output
```
Relationship status :Friends
```
~~~

Comment (silentwraith6) [reply snippets/python/flamesGame.md]: done ! Please check

**Review by kaustubhgupta — APPROVED** [review]

**Review by sakshamarora1 — APPROVED** [review]

**Pull request merged**